test/unit/spec/v1/MfaVerify_spec.js

```diff
@@ -2456,7 +2456,9 @@ Expect.describe('MFA Verify', function() {
               spyOn(SharedUtil, 'redirect');
               Util.resetAjaxRequests();
               test.setNextResponse(resCancel);
-              test.form.signoutLink($sandbox).click();
+              const $signOut = test.form.signoutLink($sandbox);
+              expect($signOut.text()).toBe('Back to sign in');
```

Comment:
Is it possible for us to add an `expect` to verify the URL is correct?

Reply:
That is being asserted farther down, line 2476